### DISCUSSION TITLE
UITextBox line-height theming?

### DISCUSSION THREAD

**Discussion by cbradburne** [MyreMylar/pygame_gui#201] [Q&A]
I may have missed how to do this but, is there any way to change the line-height of the text inside a UITextBox?

Lines of text seems a little too spaced out, maybe it's just the font I'm using. I wasn't sure if there was a theme element for line-height in a textbox.

Thanks again.

**Comment by MyreMylar** [comment]
I don't think there is anything in the current released version - though in the current build on my PC that I most familiar with now I have scrapped and rewritten all the code where that sort of thing may once have existed.

The good news is that when I finally get 0.6.0 out of the door it should have some level of line spacing feature. I'm not sure how well it will work yet as it hasn't been extensively tested - it's not been a focus, and I forget when, or why, I added it in the protracted, constantly COVID interrupted development of this version - but there is definitely a `line_spacing` parameter that gets passed into my new `TextBoxLayout` class that does something along these lines. So fingers crossed it will be useful.
